addon/services/asset-loader.js

```diff
@@ -69,6 +69,7 @@ export default Ember.Service.extend({
    * @public
    * @method loadBundle
    * @param {String} name
+   * @param {Boolean} retryLoad
```

Comment:
This was originally elided because it is only used internally and isn't part of the actual public API. That said, it's probably worth documenting it as such. So, can you add a short description stating that this is only used internally to re-initiate loads?